lib/avro_ex/schema.ex

```diff
@@ -1,5 +1,5 @@
 defmodule AvroEx.Schema do
-  alias AvroEx.{Error, Schema}
+  alias AvroEx.{Schema}
```

Comment:
Tiny nitpick for consistency with how we handle elsewhere

```suggestion
  alias AvroEx.Schema
```

Reply:
lol, funny you mention consistency, I made this change because Credo said this is the more consistent version!

I can remove the credo check, but I think its moot for now as I plan to refactor much of this

Reply:
Wow! That's kinda surprising but I trust Credo 😁